bindata/v4.1.0/kube-scheduler/pod.yaml

```diff
@@ -24,6 +24,12 @@ spec:
         sleep 1
       done
 
+      echo -n "Waiting for kube-apiserver to respond."
+      until curl --output /dev/null --silent -k https://localhost:6443/readyz; do
```

Comment:
This looks improper to me.  We switched to the internal LB for scheduler communication specifically to avoid this linkage.